mender-deb-package

```diff
@@ -2,6 +2,9 @@
 
 set -ex
 
+export DEBEMAIL=${DEBEMAIL:-"Mender root"}
+export DEBFULLNAME=${DEBFULLNAME:-"root@mender.io"}
+
```

Comment:
we were generating the `debian/changelog` file (below with dch) with root@<random-hostname-based-on-docker-container>, however devian/changelog is packaged into debian.tar.xz files, which are in turn added to dsc files, with a checksum. Now the checksum was different every time, but the debian.tar.xz files are generated per-deistribution (not per architecture and distribution): e.g.: `mender-client_3.3.0~git20220215.a905a0a-1+debian+buster+builder474683108.debian.tar`, that in turn lead to mismatched checksums while importing some packages:
```
ERROR: Unexpected content of file '/var/reprepro/incoming/mender-client_3.3.0~git20220215.a905a0a-1+debian+bullseye+builder474191641.debian.tar.xz'!
md5 expected: f80e29b7922deaef3768eaf695b682d9, got: 3083936e29d6801603d185c46afccb94
sha1 expected: 350c55457a8099568997ec55937cc54d64455b69, got: 50d7a6e57afdd178b93ca89e9db7c08f70438522
sha256 expected: 7417f752d4b841339a928650e8f1d8f664a8fae36e0405177181281cef772066, got: 9aafb01667f786f4c13a1b1819bf1635b536b05c449aa0018b27ab3b80ccb432
size expected: 5332, got: 5348
```
(it depended on the order of builds finished)